src/antsibull/data/ansible-readme.rst

```diff
@@ -56,6 +56,10 @@ Issues with plugins and modules in the Ansible package should be reported
 on the individual collection's issue tracker.
 Issues with ``ansible-core`` should be reported on
 the `ansible-core issue tracker <https://github.com/ansible/ansible/issues/>`_.
+Issues with the ``ansible`` package build process or serious bugs or
+vulnerabilities in a collection that are not addressed after opening an issue
+in the collection's issue tracker should be reported on
+`ansible-build-data's issue tracker <https://github.com/ansible-community/ansible-build-data/issues>`_.
```

Comment:
Or how about simply:
```suggestion
`ansible-build-data's issue tracker <https://github.com/ansible-community/ansible-build-data/issues>`_.
Please refer to the `ansible-build-data's README
<https://github.com/ansible-community/ansible-build-data/blob/main/README.md#issue-tracker>`_
for more details.
```

Reply:
Sounds good to me.

@gotmax23 What do you think about this? It's shorter than my original suggestion but it still refers to the README (which I think important).